.github/workflows/main.yml

```diff
@@ -29,7 +32,11 @@ jobs:
         with:
           images: aeternity/aepp-faucet
           tags: |
+            type=raw,value=latest,enable=${{ endsWith(GitHub.ref, 'master') }}
```

Comment:
if I understand it correctly this pushes latest master build as docker latest tag, while we want to push latest tagged version as docker latest tag

Reply:
okey, so only on tag , without on push in master ?

Reply:
- master branch should go to master docker tag
- vX.Y.Z tag should go to docker vX.Y.Z tag
- latest vX.Y.Z tag should go to docker `latest` tag

Reply:
From the docs , I think we need to remove these lines and will add the latest tag:
https://github.com/aeternity/aepp-faucet-nodejs/pull/8/files#diff-7829468e86c1cc5d5133195b5cb48e1ff6c75e3e9203777f6b2e379d9e4882b3R38-R39
Will make the change , but not sure how we can test within the PR

Reply:
You can test it by tagging your PR commits

Reply:
The default behaviour of metadata action is like in the table below . Isn't what we want ? I will test one more time all events : 
`

Event | Ref | Docker Tags
-- | -- | --
pull_request | refs/pull/2/merge | pr-2
push | refs/heads/master | master
push | refs/heads/releases/v1 | releases-v1
push tag | refs/tags/v1.2.3 | v1.2.3, latest
push tag | refs/tags/v2.0.8-beta.67 | v2.0.8-beta.67, latest

`

Reply:
kind of, pr-2 and releases-v1 makes no sense to me, who is gonna clean it up ?

Reply:
We currently have. only event on push in master and docket tag will be generated only if a tag is pushed/created. 
Lets we clear what we want and I will configure it.

Reply:
I'd say we want:
Event | Ref | Docker Tags
-- | -- | --
push | refs/heads/master | master
push tag | refs/tags/v1.2.3 | v1.2.3, latest
push tag | refs/tags/v2.0.8-beta.67 | v2.0.8-beta.67